week-2/L-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ var everyone = [
   "Swathi"
 ];
 
-var firstFive; // complete this statement
-var lastFive; // complete this statement
+var firstFive = everyone.slice(0, 5); // complete this statement
+var lastFive = everyone.slice(everyone.length - 6, everyone.length - 1); // complete this statement
```

Comment:
This works, but `Array.slice()` has a cool feature. It accepts a negative number and will count from the back of the array for you. So you can do this to get the last five:

```
var lastFive = everyone.slice(-5);
```